CHANGELOG.md

```diff
@@ -3,7 +3,12 @@
 ## Unreleased
 
 - [#630](https://github.com/stripe/stripe-react-native/pull/630) Fix: card field focus jumps back to the card number field on Android
-- [#824](https://github.com/stripe/stripe-react-native/pull/824) Fix: Google Pay error code when canceled should be 'Canceled' 
+- [#824](https://github.com/stripe/stripe-react-native/pull/824) Fix: Google Pay error code when canceled should be 'Canceled'
+- [#832](https://github.com/stripe/stripe-react-native/pull/832)
+  - BREAKING CHANGE: Rename `handleCardAction` to `handleNextAction`.
```

Comment:
For this I wonder if you can change the handleCardAction to be deprecated and in the annotation suggest to use handleNextAction, and then in a later release remove handleCardAction.  This will give users a little opportunity to update to the new method.

Reply:
That's a good point, I was hoping to wrap a couple different breaking changes into our v1 release, but it's def a better user experience to make this less jarring.